pkg/util/domainutil/repair_vars.go

```diff
@@ -53,6 +54,29 @@ func (r *repairInfo) GetRepairTableList() []string {
 	return r.repairTableList
 }
 
+// GetMustLoadRepairTableListByDB gets must load repair table ID list.
+func (r *repairInfo) GetMustLoadRepairTableListByDB(dbName string, tableName2ID map[string]int64) []int64 {
+	r.RLock()
+	defer r.RUnlock()
+
+	repairTableSet := make(map[string]struct{}, len(r.repairTableList))
+	for _, fullTableName := range r.repairTableList {
+		if strings.HasPrefix(strings.ToLower(fullTableName), strings.ToLower(dbName)+".") {
```

Comment:
```GetMustLoadRepairTableListByDB``` must get the lowercase dbname. So we can remove this `strings.ToLower`.

<img width="1099" alt="image" src="https://github.com/user-attachments/assets/cab0f8c6-4c77-4c03-b311-1bd355b71260" />